File: 349-intersection-of-two-arrays/intersection-of-two-arrays.js
/**
 * @param {number[]} nums1
 * @param {number[]} nums2
 * @return {number[]}
 */
var intersection = function(nums1, nums2) {
    items1 = {}
    for (let char of nums1) items1[char] = true 
    items2 = {}
    for (let char of nums2) items2[char] = true 
    let res = []
    for (let char in items1){
        if (items2[char]) res.push(Number(char))
    }
    return res
};

File: 349-intersection-of-two-arrays/intersection-of-two-arrays.py
class Solution:
    def intersection(self, nums1: List[int], nums2: List[int]) -> List[int]:
        items1 , items2, res = {}, {}, []
        for char in nums1 : items1[char] = True
        for char in nums2 : items2[char] = True
        for item in items1 : 
            if item in items2 : res.append(int(item))
        return res

        